### DISCUSSION TITLE
Unable to test and verify email settings

### DISCUSSION THREAD

**Discussion by damnshaneisthatu** [nextcloud/all-in-one#5774] [Questions] [answered]
### Steps to reproduce
1. Navigate to Basic Settings > Email Server
2. Apply SMTP, SSL, from address (Gmail address), smtp.gmail.com:465, Authentication required checked, credentials. Same exact settings from another working Nextcloud instance
3. Hit "Send email"

### Expected behavior 
"Email sent" returned confirming settings are valie.

### Actual behavior 
Error: "A problem occurred while sending the email. Please revise your settings. (Error: Email could not be sent. Check your mail server log)"

#### Host OS 
Unraid OS

**Comment by docjyJ** [comment]
Does your email account have two-factor authentication enabled?

**Reply by damnshaneisthatu** [reply]
No

**Comment by szaimen** [comment, marked as answer]
Hi, please ask this at https://help.nextcloud.com - there more eyes will see your post.
